### ISSUE TITLE
Link to accessibility posters doesn't work

### ISSUE THREAD

**Issue by kariljordan** [carpentries/instructor-training#662]
On [this page](https://carpentries.github.io/instructor-training/08-motivation/) the link to the accessibility posters doesn't work.

https://github.com/UKHomeOffice/posters/tree/master/accessibility/posters_en-UK

**Comment by fmichonneau** [comment]
correct link is here: https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf
